test/config.js

```diff
@@ -443,6 +443,9 @@ test('payloadLogFile', function (t) {
   })
 })
 
+// current agent bug where stored _encodedMetadata is never
+// refreshed.  This test passed before because sendSpan triggered
+// the initial population of _encodedMetadata
```

Comment:
Is this comment still relevant? From the way it is written, I'm not sure if this was a note to yourself while you were still looking into the issue that led to adding `updateEncodedMetadata()`.

Reply:
> Is this comment still relevant?  

Nope -- it's not.  Yoinked.